README.rst

```diff
@@ -109,8 +109,12 @@ See `CONTRIBUTING`_ for more information on how to get started.
 Community
 ---------
 
-Google Cloud Platform Python developers hang out in [Slack](https://googlecloud-community.slack.com) in the #python channel (get an invitation [here](https://gcp-slack.appspot.com/)). 
+Google Cloud Platform Python developers hang out in `Slack`_ in the #python
+channel, click here to `get an invitation`_.
 
+
+.. _Slack: https://googlecloud-community.slack.com
+.. _get an invitation: https://gcp-slack.appspot.com/
```

Comment:
If you're making the codefont change that @dhermes mentioned then I think it might be good to add a space between this line and the `License` heading as well.